core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/aggregation/AvgAggregator.java

```diff
@@ -29,7 +29,7 @@
 import static com.amazon.opendistroforelasticsearch.sql.utils.ExpressionUtils.format;
 
 /**
- * The average aggregator aggregate the value evaluated by the expression.
+ * The count aggregator aggregate the value evaluated by the expression.
```

Comment:
Is this meant for JavaDoc on `CountAggregator `?

Reply:
done.